queue/src/fiskaltrust.Middleware.Storage.MySQL/Migrations/008_QueueITuES.sqlite3

```diff
@@ -0,0 +1,41 @@
+﻿CREATE TABLE IF NOT EXISTS ftQueueES
```

Comment:
Maybe we should leave all the Spanish stuff out for now? We don't know yet if these models will look like this, I think?